src/scss/components/_feature.scss

```diff
@@ -0,0 +1,18 @@
+.feature {
+  margin: 2rem auto;
+  border-radius: 3px;
+  width: 35rem;
```

Comment:
We should try to avoid setting fixed widths, as it will not resize on smaller screens. I recommend that we set the width to 100%, so it fills the entire width of the parent. Then the parent can adjust the width of the wrapper when we get that far :)